src/main/java/frc/robot/Constants.java

```diff
@@ -58,6 +58,8 @@ public static final class DriveConstants {
 
         public static final Pose2d initialPose =
                 new Pose2d(new Translation2d(), Rotation2d.fromDegrees(90));
+
+        public static final double velocityScalar = 1.0;
```

Comment:
what does this constant mean?

Reply:
This is meant to scale the anticipation distance for shooting while moving. Sorry, I'll change it to make more sense.